snapcraft.spec

```diff
@@ -18,7 +18,7 @@ data += copy_metadata("lazr.uri")
 data += copy_metadata("wadllib")
 
 a = Analysis(
-    ["snapcraft_legacy\\cli\\__main__.py"],
+    ["snapcraft\\__main__.py"],
```

Comment:
I believe this is the correct thing to do.

This is the entrypoint for the Windows snapcraft executable.  We've migrated many commands from `snapcraft_legacy` to `snapcraft`, but `init` is the only command that is [tested](https://github.com/snapcore/snapcraft/blob/ddd328f07f1aa64c63907b9ffbd5eafa11818dde/appveyor.yml#L64) on Windows by Appveyor.

Reply:
looks good to me, but I definitely don't have the whole context of Windows snapcraft
@sergiusens do you see any harm in this?

Reply:
this is correct